src/mesh/interpolation.cxx

```diff
@@ -146,7 +146,10 @@ const Field3D interp_to(const Field3D &var, CELL_LOC loc, REGION region) {
           // coordinates
 
           Field3D var_fa = fieldmesh->toFieldAligned(var);
-          Field3D result_fa;
+          Field3D result_fa(fieldmesh);
+          if (loc != RGN_NOBNDRY) {
+            result = fieldmesh->toFieldAligned(result);
```

Comment:
and also `result_fa` rather then `result`?

Reply:
:+1: thanks @dschwoerer